Dockerfile.openshift.rhel7

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8 AS builder
```

Comment:
While you are here, why "Kubernetes Authors"?

Also, did we reference this file anywhere? If not, I would remove the `.rhel7` from its name.

Reply:
The Kubernetes Authors was left behind from the original Dockerfile. I've removed it, since this is specific to the OCP buildchain.

As far as renaming this goes, personally I think it's best to leave it as is. Unfortunately, as of this morning ART is already building it with this name. If it's important, then we can file a ticket to update their jobs...